lib/Explore/feeds/feeds.en.json

```diff
@@ -1,98 +1,105 @@
 {
-    "Nextcloud": [{
```

Comment:
I don't think the json files should be edited here.

Reply:
You're right, not sure why this was included in the linting but I reverted it